packages/client/src/views/SysAdmin/Performance/reports/operational/ApplicationsStartedReport.tsx

```diff
@@ -130,10 +130,10 @@ interface BaseProps {
   reportTimeFrom: moment.Moment
   reportTimeTo: moment.Moment
   locationId: string
+  disableFieldAgentLink?: boolean
```

Comment:
This value is actually calculated in `mapStateToProps` and returned from there. So I don't think this should be here instead of `IStateProps`.

Reply:
Good point. I think the tests are actually using properties that won't even do anything if they're passed from outside